packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -0,0 +1,127 @@
+import { ICheckboxStyles } from './Checkbox.Props';
+import {
+  ITheme,
+  mergeStyleSets,
+  getFocusStyle
+} from '../../Styling';
+import { memoizeFunction } from '../../Utilities';
+
+const MS_CHECKBOX_LABEL_SIZE = '20px';
+const MS_CHECKBOX_TRANSITION_DURATION = '200ms';
+const MS_CHECKBOX_TRANSITION_TIMING = 'cubic-bezier(.4, 0, .23, 1)';
+
+export const getStyles = memoizeFunction((
+  theme: ITheme,
+  customStyles?: ICheckboxStyles
+): ICheckboxStyles => {
+  const { semanticColors, palette } = theme;
+  const checkmarkFontColor = semanticColors.inputForegroundChecked;
+  const checkboxBorderColor = semanticColors.inputBorder;
+  const checkboxBorderHoveredColor = semanticColors.inputBorderHovered;
+  const checkboxBackgroundChecked = semanticColors.inputBackgroundChecked;
+  const checkboxBackgroundCheckedHovered = semanticColors.inputBackgroundCheckedHovered;
+  const checkboxBackgroundDisabled = semanticColors.disabledText;
+  const checkboxTextColorDisabled = semanticColors.disabledText;
+
+  const styles: ICheckboxStyles = {
+    root: [
+      getFocusStyle(theme),
+      {
+        padding: '0',
+        border: 'none',
+        background: 'none',
+        margin: '0',
+        outline: 'none',

```

Comment:
I don't think you want this outline:none here. It will interfere with the focus styles, no?